filebeat/filebeat.reference.yml

```diff
@@ -69,7 +69,7 @@ filebeat.modules:
     # can be added under this section.
     #input:
 
-#-------------------------------- Auditd Module --------------------------------
+#--------------------------------- User Module ---------------------------------
```

Comment:
I think this changed because the `title` of the first key in `fields.yml` is now `User` (see https://github.com/elastic/beats/blob/master/dev-tools/mage/config.go#L245). You'll have to change the order in that file.